### ISSUE TITLE
aws-node-rest-api-with-dynamodb deploy fails

### ISSUE THREAD

**Issue by remicastaing** [serverless/examples#109]
If I try to deploy the aws-node-rest-api-with-dynamodb example without any change, I get foolowing message:
```
Trying to populate non string value into a string for
variable ${env:DYNAMODB_TABLE}. Please make sure the
value of the property is a string.
```

**Comment by remicastaing** [comment]
Ok. I found it: just have to replace `${env:DYNAMODB_TABLE}` with `${self:provider.environment.DYNAMODB_TABLE}`.

**Comment by nikgraf** [comment]
thx, @remicastaing - interested to open a PR with a fix?

**Comment by remicastaing** [comment]
Yepp, will do it tonight, @nikgraf.

**Comment by nikgraf** [comment]
awesome, thanks @remicastaing

**Comment by remicastaing** [comment]
Oups. There is no need for a PR. It's already corrected. I was using an old copy of the serverless examples.

**Comment by remicastaing** [comment]
It was fixed in commit `a3be453` one month ago.
